docs/development/design-decisions.md

```diff
@@ -5,64 +5,81 @@ This file documents the design choices as well as configuration options.
 ## Stateless
 
 No state storage is needed on `renovate` or the source code repository apart from what you see publicly (branches, Pull Requests).
-It, therefore, doesn't matter if you stop/restart the script and would even still work if you had it running from two different locations, as long as their configuration was the same.
+It, therefore, doesn't matter if you stop/restart the script.
+The script would even still work if you had it running from two different locations, as long as their configuration was the same.
```

Comment:
The whole "State storage" thing is still worded in a confusing way to me. I don't know how I can rewrite this to be more clear... Do you have any ideas?